crates/typst-library/src/model/quote.rs

```diff
@@ -171,49 +173,51 @@ impl Show for Packed<QuoteElem> {
             let Depth(depth) = QuoteElem::depth_in(styles);
             let double = depth % 2 == 0;
 
-            // Add zero-width weak spacing to make the quotes "sticky".
-            let hole = HElem::hole().pack();
+            if !html {
+                // Add zero-width weak spacing to make the quotes "sticky".
+                let hole = HElem::hole().pack();
+                realized = Content::sequence([hole.clone(), realized, hole]);
+            }
             realized = Content::sequence([
                 TextElem::packed(quotes.open(double)),
-                hole.clone(),
                 realized,
-                hole,
                 TextElem::packed(quotes.close(double)),
             ])
             .styled(QuoteElem::set_depth(Depth(1)));
         }
 
         if block {
-            realized = BlockElem::new()
-                .with_body(Some(BlockBody::Content(realized)))
-                .pack()
-                .spanned(self.span());
+            realized = if html {
+                HtmlElem::new(tag::blockquote).with_body(Some(realized)).pack()
```

Comment:
The `blockquote` element's `cite` attribute should be set if the value of `attribution` is a single link.

Reply:
Done in e84bb4b6.